docker-compose.yml

```diff
@@ -22,8 +25,8 @@ services:
       - proxitok
     security_opt:
       - no-new-privileges:true
-    cap_drop:
-      - ALL
+    #cap_drop:
```

Comment:
Why are you commenting out this?

Reply:
It was breaking the Apache container previously. Again, the `docker-compose.yml` wasn't meant to be included in the PR.